package.json

```diff
@@ -10,14 +10,30 @@
     "download:server": "curl -o trace-compass-server.tar.gz https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/trace-compass-server-latest-linux.gtk.x86_64.tar.gz; tar -xf trace-compass-server.tar.gz",
     "start:server": "./trace-compass-server/tracecompass-server",
     "start-all:browser": "yarn start:server & yarn start:browser",
-    "start-all:electron": "yarn start:server & yarn start:electron"
+    "start-all:electron": "yarn start:server & yarn start:electron",
+    "lint": "echo lint",
+    "test": "echo test"
```

Comment:
Placeholder that you intend to replace in the future?

Reply:
Yes, @tahini is working on the linter, which the command will be used for. Unit tests is on backlog I think.